### ISSUE TITLE
Bug getting links 

### ISSUE THREAD

**Issue by jaimeiniesta** [jaimeiniesta/metainspector#144]
On metainspector 4.6.0:

```ruby
p = MetaInspector.new('http://www.adelaidezoo.com.au')
p.links.all
NoMethodError: undefined method `<<' for nil:NilClass
	from /Users/jaime/.rvm/gems/ruby-2.2.0/gems/metainspector-4.6.0/lib/meta_inspector/url.rb:92:in `rescue in parsed'
	from /Users/jaime/.rvm/gems/ruby-2.2.0/gems/metainspector-4.6.0/lib/meta_inspector/url.rb:90:in `parsed'
	from /Users/jaime/.rvm/gems/ruby-2.2.0/gems/metainspector-4.6.0/lib/meta_inspector/url.rb:77:in `with_default_scheme'
	from /Users/jaime/.rvm/gems/ruby-2.2.0/gems/metainspector-4.6.0/lib/meta_inspector/url.rb:49:in `url='
	from /Users/jaime/.rvm/gems/ruby-2.2.0/gems/metainspector-4.6.0/lib/meta_inspector/url.rb:15:in `initialize'
	from /Users/jaime/.rvm/gems/ruby-2.2.0/gems/metainspector-4.6.0/lib/meta_inspector/url.rb:61:in `new'
	from /Users/jaime/.rvm/gems/ruby-2.2.0/gems/metainspector-4.6.0/lib/meta_inspector/url.rb:61:in `absolutify'
	from /Users/jaime/.rvm/gems/ruby-2.2.0/gems/metainspector-4.6.0/lib/meta_inspector/parsers/links.rb:17:in `block in all'
	from /Users/jaime/.rvm/gems/ruby-2.2.0/gems/metainspector-4.6.0/lib/meta_inspector/parsers/links.rb:17:in `map'
	from /Users/jaime/.rvm/gems/ruby-2.2.0/gems/metainspector-4.6.0/lib/meta_inspector/parsers/links.rb:17:in `all'
	from (irb):4
	from /Users/jaime/.rvm/rubies/ruby-2.2.0/bin/irb:11:in `<main>'
```

**Comment by jaimeiniesta** [comment]
Also happens on:

http://www.monartozoo.com.au
http://www.essentialbeauty.com.au
http://www.atomix.com.au

**Comment by jaimeiniesta** [comment]
All of these sites had in common a malformed href, `http://#` that we had problems handling. This has been fixed and released as 4.6.1.
